src/generator-create-api-mesh.js

```diff
@@ -35,26 +35,30 @@ class ApiMeshCreateGenerator extends Generator {
         throw new Error('Unable to create a mesh. Run "aio plugins" and check if "@adobe/aio-cli-plugin-api-mesh" plugin is installed. Make sure you\'re using the latest version.')
       }
     }
-
     this.log('Checking if selected workspace doesn\'t have a mesh')
     let shouldCreateMesh = false
     try {
-      await this.spawnCommand('aio', ['api-mesh', 'get'], { stdio: [process.stderr] })
+      const output = await this.spawnCommand('aio', ['api-mesh', 'get'], { stdio: [process.stderr] })
+      // The command might produce output on stderr without failing
+      if (output.stderr && output.stderr.includes('No mesh found')) {
+        shouldCreateMesh = true
+      }
     } catch (err) {
       /* istanbul ignore next */
+      // If the command fails, check if the error message indicates no mesh was found
       if (err.stderr.includes('No mesh found')) {
-        // no mesh in workspace so command fails
         shouldCreateMesh = true
+      } else {
+        this.log('unexpected error occurred while getting mesh: ' + err)
```

Comment:
```suggestion
        this.error('An unexpected error occurred while getting the mesh.')
```
Let's omit the actual error object here. If this error is encountered, it means there was an issue performing the get command. It's unlikely that knowing the precise error will enable the user to fix this on their end.